xc7/primitives/bufr/ntemplate.bufrN.pb_type.xml

```diff
@@ -0,0 +1,63 @@
+<!-- Model of BUFR group in HCLK_IOI3 -->
+<pb_type name="BUFR{N}" num_pb="1" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <clock name="I" num_pins="1"/>
+  <input name="CLR" num_pins="1"/>
+  <input name="CE" num_pins="1"/>
+  <output name="O" num_pins="1"/>
+
+  <mode name="BYPASS">
```

Comment:
This split seems weird, can you point to the documentation for it working this way (e.g. that CE/CLR do not exist and are not used?)

Reply:
I'll provide link to the documentation tomorrow.

Basically the BUFR has the BYPASS mode and "divide by 1" mode. In the former the signal just passes through asynchronously. So it does not use CE and CLR. In the latter I'm not sure what's going on.

The documentation says that those two modes differ in timings. The only way of controlling the timing through a verilog parameter is to make different leaf pb_types and map into them in the techmap accordingly.

Reply:
@litghost https://www.xilinx.com/support/documentation/user_guides/ug472_7Series_Clocking.pdf, page 52.

Reply:
But that documentation does not say that the ce pin doesn't work with bypass.  I believe you need all pins for the bypass black box